report/renderer/performance-category-renderer.js

```diff
@@ -167,6 +167,24 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
     return 'diagnostic';
   }
 
+  /**
+   * @param {LH.ReportResult.AuditRef[]} metrics
+   */
+  _reorderMetrics(metrics) {
+    const col1 = metrics
```

Comment:
```suggestion
    metrics = [...metrics];
    const col1 = metrics
```

nit: I had trouble understanding how this worked, forgetting that splice modified the input of the array. Should clone the input so it isn't destructive (doesn't actually change runtime here); but it is good to denote that this array reference is being mutated.

Reply:
+1, it's a nit but agreed, `splice` does make me think a bit too hard here. Another options is something like

```js
const halfway = Math.floor(metrics.length / 2);
const col1 = metrics.slice(0, halfway).reverse();
const col2 = metrics.slice(halfway).reverse();
```